### DISCUSSION TITLE
FT4233HPQ Support

### DISCUSSION THREAD

**Discussion by CFaldon** [eblot/pyftdi#275] [Ideas]
Is the FT4233HPQ package supported by PyFTDI?

**Comment by eblot** [comment]
I never tried... I guess that the 4233 uses a new bcdDevice id, so PyFtdi needs to be upgraded to support this id - at least.

**Reply by CFaldon** [reply]
Is support planned to be implemented at any point?

**Reply by eblot** [reply]
It would be nice, but I first need to find the HW device... Is it available as a module or as a cable somewhere?

**Reply by CFaldon** [reply]
There is an unreleased evaluation module: https://ftdichip.com/products/umft4233hpev/
I'm still awaiting confirmation from FTDI on the expected release date of this

**Reply by eblot** [reply]
Please let me know when this HW is available.

**Reply by CFaldon** [reply]
From FTDI sales - "UMFT4233HPEV will be launching in Q1-2022 around March".

**Comment by bm16ton** [comment]
Hello, I have one of these eval boards and am willing to try and help/test with it. Unfortunately im useless with python and use kernel drivers for i2c spi etc, but this board requires more eeprom options for the PD control and extra gpio pins etc. and none of the linux/opensource  eeprom programming apps have support yet. But im down to test whatever if it helps! Unfortunately resources for bare metal programming my drivers essentially dont exist, so being a part of community to share what ive learned and to get more info sounds amazing!

**Comment by eblot** [comment]
https://www.mouser.fr/ProductDetail/FTDI/UMFT4233HPEV?qs=Rp5uXu7WBW%2FdaOCe7Nr6DQ%3D%3D

Above 100€ for a useless evaluation board, and it seems there is no mini module option for now...

Hopefully it the same kind of device as [FT232HPQ](https://github.com/eblot/pyftdi/discussions/371), such as 4232H relates to 232H.
